pkg/interface/src/views/apps/permalinks/TranscludedNode.tsx

```diff
@@ -20,7 +20,7 @@ function TranscludedLinkNode(props: {
   api: GlobalApi;
 }) {
   const { node, api, assoc, transcluded } = props;
-  const idx = node.post.index.slice(1).split('/');
+  const idx = node?.post?.index?.slice(1).split('/') ?? [];
```

Comment:
`.slice(1).split()` will also error because .slice will return undefined and we'll attempt to access split on it.

Reply:
You want every single access to be forgiving of `undefined` and then let `??` catch